src/flownet/network_model/_mitchell.py

```diff
@@ -94,13 +93,26 @@ def mitchell_best_candidate_modified_3d(
             z_hull = z
 
         # Determine the convex hull of the original or linearly scaled perforations
-        if np.all(z == z[0]):
+        if np.isclose(z_min, z_max):
             # 2D cases
             perforation_hull = Delaunay(np.column_stack([x_hull, y_hull]))
         else:
             # 3D cases
             perforation_hull = Delaunay(np.column_stack([x_hull, y_hull, z_hull]))
 
+    if mitchell_mode == "fast":
+        x_candidate, y_candidate, z_candidate = _generate_candidates(
+            num_candidates,
+            x_min,
+            x_max,
+            y_min,
+            y_max,
+            z_min,
+            z_max,
+            perforation_hull,
+            concave_hull_bounding_boxes,
+        )
```

Comment:
You are writing this list of arguments twice in the code. To save some space you could write:

```
arguments = { "arg1": 1, "arg2": 2}
function(**arguments)
```

Reply:
I could do this. But then the function itself becomes a bit unclear with just **arguments. I would prefer to keep it this way to have the function complete with all the keywords listed.